pkgs/by-name/po/porn-vault/package.nix

```diff
@@ -0,0 +1,112 @@
+{
+  fetchFromGitLab,
+  fetchurl,
+  lib,
+  pnpm,
+  stdenvNoCC,
+  nodejs_22,
+  ffmpeg,
+  imagemagick,
+  writeShellScript,
+  makeWrapper,
+  autoPatchelfHook,
+}:
+let
+  izzy = stdenvNoCC.mkDerivation rec {
+    pname = "izzy";
+    version = "2.0.2";
+
+    src = fetchurl {
+      url = "https://gitlab.com/porn-vault/izzy/-/package_files/151366122/download";
+      hash = "sha256-K+rVb2F3HizMKIpY1zbLj5zE7B6DkIoq8nJYPUpQn+Y=";
+    };
+
+    dontUnpack = true;
+
+    nativeBuildInputs = [
+      autoPatchelfHook
+    ];
+
+    installPhase = ''
+      install -Dm755 $src $out/bin/izzy
+    '';
+
+    meta = with lib; {
+      description = "Rust In-Memory K-V Store with Redis-Style File Persistence and Secondary Indices";
+      homepage = "https://gitlab.com/porn-vault/izzy";
+      license = lib.licenses.gpl3;
+      maintainers = with maintainers; [ luNeder ];
+      platforms = [ "x86_64-linux" ];
```

Comment:
Nit: maybe add `meta.sourceProvenance`?

Reply:
That's for when you're fetching a binary, right? I was able to compile izzy so that'll no longer be needer, right?

Reply:
If you can compile izzy from source, yes :)